Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -138,7 +138,12 @@ def bump_formula_pr
     new_url = ARGV.value("url")
     if new_url && !formula
       is_devel = ARGV.include?("--devel")
-      base_url = new_url.split("/")[0..4].join("/")
+      new_url_split = new_url.split("/")
+      base_url = if new_url_split.count > 5
+        new_url_split[0..4].join("/")
+      else
+        new_url_split[0..(new_url_split.count - 2)].join("/")
```

Comment:
With all these splits a comment would be cool explaining what's going on here (as I have no idea). Perhaps using some/a new variables for the `5`, `4`, `new_url_split.count - 2`?